### DISCUSSION TITLE
Can't install Ruby 2.6.5 on Mac 10.14.6

### DISCUSSION THREAD

**Discussion by efertsch** [rbenv/ruby-build#1951] [Build failures]
I am setting up a new Mac and I am unable to install Ruby 2.6.5.

I am running macOS Mojave 10.14.6, iTerm2 build 3.3.6, Homebrew 2.1.16, rbenv 1.1.2, ruby-build 20191111.

Here is the output I have been getting:

rbenv install 2.6.5
Downloading openssl-1.1.1d.tar.gz...
-> https://dqw8nmjcqpjn7.cloudfront.net/1e3a91bc1f9dfce01af26026f856e064eab4c8ee0a8f457b5ae30b40b8b711f2
Installing openssl-1.1.1d...
Installed openssl-1.1.1d to /Users/efertsch/.rbenv/versions/2.6.5

Downloading ruby-2.6.5.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.5.tar.bz2
error: failed to download ruby-2.6.5.tar.bz2

BUILD FAILED (OS X 10.14.6 using ruby-build 20191111)

Inspect or clean up the working tree at /var/folders/2_/p202cjsn7fbdtn14v1fk6mqh0000gn/T/ruby-build.20191119101615.74488.mC7skZ
Results logged to /var/folders/2_/p202cjsn7fbdtn14v1fk6mqh0000gn/T/ruby-build.20191119101615.74488.log

Last 10 log lines:
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/scrypt.html
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/SM2.html
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/ssl.html
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/X25519.html
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/X448.html -> /Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/X25519.html
/Users/efertsch/.rbenv/versions/2.6.5/openssl/share/doc/openssl/html/man7/x509.html
/var/folders/2_/p202cjsn7fbdtn14v1fk6mqh0000gn/T/ruby-build.20191119101615.74488.mC7skZ ~/Desktop
~/Desktop
/var/folders/2_/p202cjsn7fbdtn14v1fk6mqh0000gn/T/ruby-build.20191119101615.74488.mC7skZ ~/Desktop
curl: (6) Could not resolve host: cache.ruby-lang.org

I have tried a number of things:
1. brew update && brew upgrade ruby-build
2. Uninstalling and re-installing rbenv
3. Unistalling and re-installing openssl
4. Unlinking and re-linking pkg-config
5. This command: RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl)" rbenv install  2.6.5
6. This command: CC=clang rbenv install 2.6.5

Can't seem to get anything working. [This issue](https://github.com/rbenv/ruby-build/issues/1320) makes it seem like it could be a network error?

Any and all help is greatly appreciated!

**Comment by efertsch** [comment]
Power cycling is an obvious, yet beautiful gift.

**Comment by stwr667** [comment]
> Power cycling is an obvious, yet beautiful gift.

Yep, restarting fixed this for me too. On MacOS 10.15.1. ruby 2.6.5.

**Comment by grisaitis** [comment]
Adding `--verbose` / `-v` might help (`rbenv install --verbose 2.6.5`)

**Comment by philippb** [comment]
For me updating `wget` with `brew install wget` did the trick
